source/seismology/seismic-source.md

```diff
@@ -57,15 +57,35 @@
 将大量地震事件的参数进行汇总整理得到的目录资料，称之为地震目录。地震目录对于研究
 地震活动性、地震灾害评估等具有重要的意义。
 
-地震目录中可以包含地震的很多参数，其中至少应包括地震的时空强信息，即：
+地震目录至少应包含地震的时空强信息，即：
 
 - 发震时刻
 - 震中经纬度
 - 地震深度
 - 地震震级
 
-有些地震目录中，还提供了地震破裂时间、地震震源机制等其他参数。
+除此之外，有些地震目录还提供了地震的破裂时间和震源机制等其他参数。
 
-根据所包含地震的区域范围，地震目录可以分成两类：全球地震目录和区域地震目录。顾名思义，
-全球地震目录包含了全球范围内的地震，但通常只包含比较大的地震，小地震多有遗漏；
-区域地震目录则只关注某个特定区域内的地震，一般包含更多更小的地震。
\ No newline at end of file
+根据所包含地震的区域范围，地震目录可以分成两类：全球地震目录和区域地震目录。
+顾名思义，全球地震目录包含了全球范围内的地震，但通常只包含比较大的地震，小地震多有遗漏；
+区域地震目录则只关注某个特定区域内的地震，通常包含更多的小地震。
+
+常见的全球地震目录包括：
+
+ISC 地震目录
+: [ISC](http://www.isc.ac.uk/iscbulletin/) (International Seismological Center)
+  地震目录是全球地震的最终版本，其涵盖了 1900 年至今的地震信息。所有的地震信息
+  通过程序自动确定，并由 ISC 的分析人员对其进行人工检查和重定位。经过人工审核的
+  ISC 地震目录通常有 24 个月的延迟。
+
+USGS PDE 地震目录
+: PDE 全称是 Preliminary Determination of Epicenters，是由美国地质调查局提供的
+  地震快速定位结果。PDE 地震目录包含了 1960 年至今的地震信息，其确定地震信息是
+  全自动且近实时的。
```

Comment:
I think [ANSS Comprehensive Earthquake Catalog](https://earthquake.usgs.gov/data/comcat/) (ComCat) is now a more popular catalog than USGS PDE. ComCat is maintained by [ANSS](https://www.usgs.gov/programs/earthquake-hazards/anss-advanced-national-seismic-system), which includes a national Backbone network, the National Earthquake Information Center ([NEIC](https://www.usgs.gov/programs/earthquake-hazards/national-earthquake-information-center-neic)), the National Strong Motion Project, and 15 regional seismic networks operated by USGS and its partners. Meanwhile, USGS PDE is maintained by NEIC, and is also a contributing catalog for ComCat. See https://earthquake.usgs.gov/data/comcat/index.php#1:

<img width="1292" alt="" src="https://user-images.githubusercontent.com/50591376/179390570-de8bd22c-483c-433a-bac9-2f28efad6964.png">